packages/commonwealth/client/scripts/views/components/Privy/dialogs/CodeDialog.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+type CodeDialogProps = {
+  onVerify: (code: string) => void;
+  onCancel: () => void;
+};
+
+/**
+ * Background dialog that we run along with the store so that we can finish auth.
+ */
+export const CodeDialog = (props: CodeDialogProps) => {
```

Comment:
Privy does not provide their UI for confirmation code? It was part of the flow in magic

Reply:
They do not unfortunately :-/